<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_stage</class>
 <widget class="QWidget" name="add_stage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Karla&quot;;
</string>
   </property>
   <property name="text">
    <string>Название стадии</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: #FF5B5B;
	color: white;
}
QPushButton:pressed {
	background-color: #FFBCBC;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #6D55FF;
	border-radius: 15px; 
	font: 75 25pt &quot;Karla&quot; bold;
	border: 2px;
}
QPushButton:pressed{
	background-color: #A597FF;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: #45FF79;
	color: white;
}
QPushButton:pressed {
	background-color: #B2FFC8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="greeting.qrc">
     <normaloff>:/new/prefix1/return.png</normaloff>:/new/prefix1/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px; 
font: 14pt &quot;Karla&quot;;
padding-left: 10px;
padding-right:10px;
border: 2px solid #6D55FF;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(109, 85, 255);
font: 75 40pt &quot;Karla&quot; bold;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>Добавить стадию</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Karla&quot;;
</string>
   </property>
   <property name="text">
    <string>Описание стадии</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Karla&quot;;
</string>
   </property>
   <property name="text">
    <string>Рекомендуемая длина(часы)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px; 
font: 14pt &quot;Karla&quot;;
padding-left: 10px;
padding-right:10px;
border: 2px solid #6D55FF;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Karla&quot;;
</string>
   </property>
   <property name="text">
    <string>Сложность стадии</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	border-radius: 15px; 
	font: 14pt &quot;Karla&quot;;
	padding-left: 10px;
	padding-right:10px;
	border: 2px solid #6D55FF;
}
QComboBox::drop-down {
        subcontrol-origin: padding;
        subcontrol-position: top right;
        width: 20px;
        border-left-width: 1px;
        border-left-color: darkgray;
        border-left-style: solid;
        border-top-right-radius: 3px;
         border-bottom-right-radius: 3px;
}
QComboBox::down-arrow {
        image: url(:/new/prefix1/combobox.png);
		width:20px;
		height:20px;
}
QComboBox QAbstractItemView {
    border: 2px solid #6D55FF;
    color: #6D55FF;
    background-color: white; 
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px; 
font: 14pt &quot;Karla&quot;;
padding-left: 10px;
padding-right:10px;
border: 2px solid #6D55FF;
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="greeting.qrc"/>
 </resources>
 <connections/>
</ui>
